12/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/rte-blocks.md

```diff
@@ -0,0 +1,97 @@
+# Blocks in Rich Text Editor
+
+It is possible to insert Blocks into the markup of the Rich Text Editor (RTE). Once you've defined the Block Type as part of the RTE Data Type and enabled the Blocks Toolbar Option. Then Blocks can be created directly inside the Rich Text Editor.
+
+## Configure Blocks
+
+The Block List property editor is configured in the same way as any standard property editor, via the _Data Types_ admin interface.
+
+To set up your Block List Editor property, create a new _Data Type_ and select **Block List** from the list of available property editors.
+
+Then you will see the configuration options for a Block List as shown below.
+
+![Rich Text Editor - Data Type Block Fields](./images/rte-data-type-block-fields.jpg)
+
+The Data Type editor allows you to configure the following properties:
+
+* **Available Blocks** - Here you will define the Block Types to be available for use in the property. Read more on how to set up Block Types below.
+* **Blocks Live editing mode** - Enabling this will make editing of a Block happening directly to the Rich Text Editor, making changes appear as you type.
+
+## Setup Block Types
+
+Block Types are **Element Types** which need to be created before you can start configuring them as Block Types. This can be done either directly from the property editor setup process, or you can set them up beforehand and add them to the Rich Text Editor after.
```

Comment:
```suggestion
Block Types are **Element Types** which need to be created before you can start configuring them as Block Types. This can be done either directly from the property editor setup process, or you can set them up beforehand. If they are set beforehand then they need to be added to the Rich Text Editor afterward.
```